doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,382 @@
+********
+FABRID
+********
+.. _fabrid-design:
+
+- Author: Justin Rohrer, Jelte van Bommel, Marc Odermatt, Marc Wyss, Cyrill Krähenbühl
```

Comment:
Please also add Juan here -> he will also have to make a few iterations over this document